packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -67,10 +67,17 @@ export const chatMessageDateStyle = mergeStyles({
 /**
  * @private
  */
-export const chatMessageAuthorStyle = mergeStyles({
-  fontWeight: FontWeights.semibold,
-  fontSize: '0.75rem'
-});
+export const chatMessageAuthorStyle = mergeStyles({ fontWeight: FontWeights.semibold, fontSize: '0.75rem' });
+
+/**
+ * @private
+ */
+export const chatMessageDateFailedStyle = (theme: Theme): string =>
+  mergeStyles({
+    color: theme.palette.neutralPrimaryAlt,
```

Comment:
Could you also update `color` in `chatMessageDateStyle` to use `theme.tokens` instead of `tokens` as `chatMessageDateStyle` is v8 style but `tokens` is v9 api?

Not sure if in this case we still need `chatMessageDateFailedStyle`

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/73612854/1a8c158b-b5f1-4b28-afff-b3b39a4838c4)

Reply:
I tested hover, it appears to work but for consistency, maybe we should remove both? If we can test and it works.

Reply:
`chatMessageDateFailedStyle` and `chatMessageDateStyle` have different color. So we still need both of them